src/utils.js

```diff
@@ -115,3 +117,13 @@ export function partition(array, n) {
     return acc;
   }, []);
 }
+
+/**
+ * Checks if a string is an UUID, according to RFC4122.
+ *
+ * @param  {String} uuid The uuid to validate.
+ * @return {Boolean}
+ */
+export function isUUID(uuid) {
```

Comment:
That's not what wikipedia says https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_.28random.29